source/fuzz/fuzzer_util.cpp

```diff
@@ -364,6 +364,16 @@ bool IsCompositeType(const opt::analysis::Type* type) {
                   type->AsVector());
 }
 
+bool IsValidConstant(opt::IRContext* ir_context, uint32_t id) {
+  return ir_context->get_def_use_mgr()->GetDef(id)->opcode() == SpvOpConstant;
+}
+
+bool IsZeroConstant(opt::IRContext* ir_context, uint32_t id) {
+  auto inst = ir_context->get_def_use_mgr()->GetDef(id);
+  return (inst->opcode() == SpvOpTypeInt || inst->opcode() == SpvOpTypeFloat)
+            && inst->GetSingleWordOperand(1) == 0;
+}
```

Comment:
We usually put stuff into `fuzzerutil::` if it's used by multiple transformations. If it's only used in a single one, it might be better to keep it in that transformation's scope.

Reply:
I realized that they were used in an earlier version of implementation and no longer needed. Removed!